Mesh_3/CMakeLists.txt

```
@@ -0,0 +1,16 @@
+# This is the CMake script for compiling this folder.
+
+cmake_minimum_required(VERSION 3.1...3.23)
+project(Mesh_3)
+
+if(CGAL_ENABLE_TESTING)
+  enable_testing()
+endif()
+
+option(CGAL_MESH_3_BENCHMARK "Add benchmark to project" OFF)
+if(CGAL_MESH_3_BENCHMARK)
+  add_subdirectory("benchmark/Mesh_3")
+endif()
+
+add_subdirectory("examples/Mesh_3")
+add_subdirectory("test/Mesh_3")
```

To make it more copy-pastable:

```suggestion
option(CGAL_MESH_3_BENCHMARK "Add benchmark to project" OFF)
if(CGAL_MESH_3_BENCHMARK)
  add_subdirectory("benchmark/${PROJECT_NAME}")
endif()

add_subdirectory("examples/${PROJECT_NAME}")
add_subdirectory("test/${PROJECT_NAME}")
```

I do not know what to do with benchmarks.